test/erddap_test.js

```diff
@@ -1,25 +1,43 @@
 /* eslint-disable no-undef */
 const chai = require('chai');
 const chaiHttp = require('chai-http');
-const app = require('../app/server');
+const app = require('@/server');
 
 chai.use(chaiHttp);
 chai.should();
+// const expect = chai.expect; 
 
 describe('Buoy', () => {
-  describe('GET /', () => {
-    // Test to get all students record
+  describe('GET /erddap/buoy', () => {
```

Comment:
should be `/erddap/buoy/query`?

Reply:
it should, but I actually think we should just remove the `query` word from the route

Reply:
why? I think the query makes a more idiomatic api

Reply:
`?` means `query` already, it's just repetitive and makes the url longer.

Reply:
but `/?kdljfaoweifa;soefh` looks funny (and I think I tried it and it didn't work)

Reply:
I'm confused, what did you try?
It should be just:

`/erddap/buoy?ids=1,2&variables=foo,bar"

The things after the `?` are query parameters.

Reply:
but because everything is mounted under `/buoy`, `/` refers to `/buoy/`